### PR TITLE
docs: fix broken links

### PR DESCRIPTION
**Pull request by milasuperstar**
fixes the broken links after we restructured the docs section.

CleanShot 2022-07-14 at 11 25 55

Signed-off-by: Mila Wu <wumila1996@gmail.com>

### CODE REVIEW

File: content/docs/en/reference/command-line.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Because --host is part of the webhook callback endpoint. If the same Bytebase in
 
 default: ""
 
-Optional external PostgreSQL instance connection url. If specified, Bytebase will store its own metadata there. Check [setup guide](/docs/get-started/deploy-to-production/external-postgres) for details.
+Optional external PostgreSQL instance connection url. If specified, Bytebase will store its own metadata there. Check [setup guide](https://www.bytebase.com/docs/get-started/install/external-postgres) for details.
```
Comment (boojack) [inline content/docs/en/reference/command-line.md:78]: Maybe we shouldn't add the `https://www.bytebase.com` prefix.

**Review by boojack — COMMENTED** [review]
LGTM otherwise

**Review by boojack — APPROVED** [review]
LGTM

image

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**